.github/workflows/tests_linux_cpp.yml

```diff
@@ -0,0 +1,305 @@
+name: Testing (Linux, C++)
```

Comment:
Question: do we need to split the file into 2? I think we should keep the same single source as the others, even if it is a little long. We could add some labeling and comment structuring though if you'd like. Also, splitting like this erases all our history on that other file, so I'd prefer to avoid it and make it easier to track back on the decisions we have in there.

I think if you would prefer to split it we can add all the original authors as co-authors to the commit (same as with Lightning Kokkos), and a reference to the old name to track through the git history.

Which would you prefer?

Reply:
I revoked the split and combined all Linux tests in a single file as it was.